nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/core/im/IMImageUtils.java

```diff
@@ -124,6 +124,11 @@ protected void makeFiles(Blob blob, String targetExt)
                 targetExt = ext;
             }
             targetFile = Framework.createTempFile("nuxeoImageTarget", "." + targetExt);
+
+            Framework.trackFile(targetFile, targetFile);
+            if (tmpFile != null) {
+                Framework.trackFile(tmpFile, tmpFile);
```

Comment:
`tmpFile` is already deleted in the `finally` block of `call()`, so no need to track it.